src/js/lib/monster.util.js

```diff
@@ -81,9 +81,26 @@ define(function(require) {
 		},
 		unformatPhoneNumber: unformatPhoneNumber,
 		unixToDate: unixToDate,
-		updateImagePath: updateImagePath
+		updateImagePath: updateImagePath,
+		isAuthorizedTopicForCrossSiteMessaging: isAuthorizedTopicForCrossSiteMessaging
 	};
 
+	function isAuthorizedTopicForCrossSiteMessaging() {
+		var TopicForCrossSiteMessaging = [
+			'voip.tab.myOffice',
```

Comment:
Is there a reason to use a topic name that is different to the ones that the app already handles?

Reply:
Apologies again but that was done based on RFC that @joristirado passed on.